src/oInput/oInput.js

```diff
@@ -0,0 +1,98 @@
+import mapEvents from '../utils/eventMapper';
+import './input.css';
+import o from 'ojs-core';
+
+const TYPES = {
+    text: 'Text field',
+    date: 'Date field',
+    email: 'E-mail field',
+    month: 'Month field',
+    number: 'Number field',
+    password: 'Password field',
+    tel: 'Tel field',
+    file: 'File field',
+    url: 'URL field',
+};
+
+class oInput {
+    constructor(config) {
+        this.store = {
+            label: '',
+            type: 'text',
+            typeText: false,
+            placeholder: '',
+            required: true,
+            index: false,
+            labelClass: 'input__label',
+            labelStyle: false,
+            inputClass: 'input__field',
+            inputStyle: false,
+            attributes: [],
+        };
+
+        this.configMerge(config);
+        this.store.events = this.events(config);
+        this.store.id = config.id ? `${config.id}--input` : `${new Date().getTime()}${Math.floor(Math.random() * 100)}--input`;
```

Comment:
Wrzuć to może do osobnej, zewnętrznej metody, która swoją nazwą by trochę uprościła w czytaniu to.

Reply:
chodzi o to dodawanie hasha do id (new Date()...)